terratorch/models/pixel_wise_model.py

```diff
@@ -7,7 +7,7 @@
 
 from terratorch.models.heads import RegressionHead, SegmentationHead
 from terratorch.models.model import AuxiliaryHeadWithDecoderWithoutInstantiatedHead, Model, ModelOutput
-
+from terratorch.models.backbones.prithvi_vit import pad_images
```

Comment:
Maybe we can move it to an model utils script instead of having it in a prothvi script?